godaddyddns/godaddy.py

```diff
@@ -85,6 +89,10 @@ def update_a_record(self, domain: str, ip: str, ttl: int = 600) -> None:
             headers=self.headers,
             json=[{"data": ip, "ttl": ttl}],
         )
-        self.printer(f"{response.request.url} {response.status_code}")
+        logger.debug(
+            "Request url: {}, Status Code: {}".format(
```

Comment:
Please use f strings as per Google style docs

Reply:
e.g. logger.debug(f"Request url: {response.request.url}, Status Code: {response.status_code}")

I guess whilst at it, url should be capitalised as well.